recipes/libassert/all/conandata.yml

```diff
@@ -0,0 +1,32 @@
+sources:
+  # Newer versions at the top
+  "1.1":
+    url:
+      - "https://github.com/jeremy-rifkin/libassert/archive/refs/tags/v1.1.tar.gz"
+    sha256: "01e1b99bf4dacd816fceeb5cc7efdc8875ed21295a62396584a0e7649b944c67"
+  "1.0":
+    url:
+      - "https://github.com/jeremy-rifkin/libassert/archive/refs/tags/v1.0.tar.gz"
+    sha256: "e1bb3b50767994ca4d0f60b7977b279cf32b8569ff92c5830e7a1de567b82fd5"
+
+patches:
+  "1.1":
+    - patch_file: "patches/1.1/0001-fix-max_line_number_width-miscalculation.patch"
+      patch_type: "bugfix"
+      patch_description: "Fix max_line_number_width miscalculation"
+    - patch_file: "patches/1.1/0002-binary_diagnostics_descriptor-noexcept.patch"
+      patch_type: "portability"
+      patch_description: "Fixes GCC 8 build"
+    - patch_file: "patches/1.1/0003-cmake-windows.patch"
+      patch_type: "bugfix"
+      patch_description: "Fixes Windows DLL CMake install"
+  "1.0":
+    - patch_file: "patches/1.0/0001-add-cstring-include.patch"
+      patch_type: "bugfix"
+      patch_description: "Add missing include"
+    - patch_file: "patches/1.0/0002-cmake-backport.patch"
+      patch_type: "portability"
+      patch_description: "Port back CMake install"
+    - patch_file: "patches/1.0/0003-fix-max_line_number_width-miscalculation.patch"
+      patch_type: "bugfix"
```

Comment:
Quick question: are these patches official or custom made for Conan?

Reply:
The miscalculation bug is already merged back and will therefore be part of the next release, the other 1.1 patches will soon follow.

Other than that I'll have to check with Jeremy :)